### ISSUE TITLE
Providing website feedback

### ISSUE THREAD

**Issue by alexjuiceman** [voteflux/flux-website-v2#110]
The current process of providing feedback or suggesting changes for the website feels very daunting/complicated to someone without a background in coding. It was difficult initially to figure out who was responsible or where to go to find out, and even once finding this GitHub it took multiple questions in the Discord to find the place to actually suggest changes.

If this is the way you would prefer feedback about the site to be submitted, it would be great to have that clearly laid out somewhere accessible. If there is any easier method that would be reasonable to adopt that would be great.

I read the contributing guidelines for this project but I have no idea what they mean so I hope this is an acceptable format.

**Comment by XertroV** [comment]
> The current process of providing feedback or suggesting changes for the website feels very daunting/complicated to someone without a background in coding

Yeah, not ideal. I've thought of adding an "edit on github" or "talk on github" type link as you sometimes see around (which would take you to the "create an issue" page or something)

> If this is the way you would prefer feedback about the site to be submitted, it would be great to have that clearly laid out somewhere accessible

Here is good. Despite my slow response I did see your issue back in april. It also allows anyone to "watch" the repo and get notifications if someone opens an issue.

Open to suggestions on how to make that clear or easier tho.

> I read the contributing guidelines for this project but I have no idea what they mean so I hope this is an acceptable format.

Yeah, this is totally fine. contribution guidelines are particularly for code (usually)
